specification/vi/resource-manager/Microsoft.VideoIndexer/preview/2022-07-20-preview/vi.json

```diff
@@ -0,0 +1,1396 @@
+        "description": "Generate an Azure Video Indexer restricted viewer access token.",
+        "x-ms-examples": {
+          "Generate restricted viewer accessToken for video #1": {
+            "$ref": "./examples/ViGenerateRestrictedViewerAccessToken/ViGenerateVideoRestrictedViewerAccessToken1.json"
+          },
+          "Generate restricted viewer accessToken for video #2": {
+            "$ref": "./examples/ViGenerateRestrictedViewerAccessToken/ViGenerateVideoRestrictedViewerAccessToken2.json"
+          },
+          "Generate restricted viewer accessToken for project #1": {
+            "$ref": "./examples/ViGenerateRestrictedViewerAccessToken/ViGenerateProjectRestrictedViewerAccessToken1.json"
+          },
+          "Generate restricted viewer accessToken for project #2": {
+            "$ref": "./examples/ViGenerateRestrictedViewerAccessToken/ViGenerateProjectRestrictedViewerAccessToken2.json"
+          },
+          "Generate restricted viewer accessToken for account": {
+            "$ref": "./examples/ViGenerateRestrictedViewerAccessToken/ViGenerateAccountRestrictedViewerAccessToken.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "parameters",
+            "x-ms-client-flatten": true,
+            "description": "The parameters for generating restricted viewer access token",
+            "schema": {
+              "$ref": "#/definitions/GenerateRestrictedViewerAccessTokenParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Azure Video Indexer account's access token properties were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/AccessToken"
+            }
+          },
+          "400": {
+            "description": "Bad Request.",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
```

Comment:
Generally, you don't need to explicitly add these responses, these are covered under "default"

Reply:
I understand, I want to stay with this convention as this new API version which only adds a single new API. 
This API is extremely similar to "GenerateAccessToken" API and we wish the doc to look the same (examples. responses etc.)